index.html

```diff
@@ -6,24 +6,35 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>LinuxChix India</title>
-    <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arima+Madurai">
```

Comment:
May I know the reason for removing this font?

If it is removed, have we tried to fix the changes in `main.css` and anywhere else that it is currently being used?

Reply:
This seems to have been missed. Really sorry.
Shall I raise another PR with just the `keywords`?

Reply:
You can do the changes here itself.